app/models/salt_handler/minion_start.rb

```diff
@@ -20,12 +20,14 @@ def initialize(salt_event)
   def process_event
     parsed_data = salt_event.parsed_data
 
-    # Ignore the ca minion. It shouldn't be used as part of the k8s cluster.
-    return false if parsed_data["id"] == "ca"
+    # Ignore the ca and the dashboard minions. It shouldn't be used as part of
+    # the k8s cluster.
+    id = parsed_data["id"]
+    return false if id == "ca" || id == "admin"
```

Comment:
This could be simplified for future values as `return false if ["ca", "admin"].include? id`